lib/danger/request_source/bitbucket_server_api.rb

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+require "danger/helpers/comments_helper"
+
+module Danger
+  module RequestSources
+    class BitbucketServerAPI
```

Comment:
I would expect this in here, it's the main way in which Danger communicates with BitBucket server? It's the pattern we use for github/gitlab too